### PR TITLE
Fixes the AI core being flammable

### LINKED ISSUE #28737: AI core can catch on fire, yet the fire is not visible.
This means that running into the core during this will set you on fire, with no visual tell.

### PR DESCRIPTION
**Pull request by silicons**
>plasteel and rglass, supposedly extremely strong
>flammable ai core

fixes #28737

### CODE REVIEW

**Comment by KingPhilipIII** [comment]
Feature.

**Comment by Kraseo** [comment]
cyborgs are also flammable but i dont see complaints about that KEVINZ

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -164,6 +164,8 @@
 
 	. = ..()
 
+/mob/living/silicon/ai/IgniteMob()
+	fire_stacks = 0
```
Comment (Cyberboss) [inline code/modules/mob/living/silicon/ai/ai.dm]: add . = ..() after, nothing should happen because fire_stacks are 0

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Comment by lzimann** [comment]
cyborgs being flammable is intended afaik

**Pull request merged**